ui/pages/confirmations/components/confirm/row/typed-sign-data/typedSignData.tsx

```diff
@@ -27,11 +27,14 @@ export const ConfirmInfoRowTypedSignData = ({ data }: { data: string }) => {
   const { sanitizedMessage, primaryType } = parseMessage(data);
 
   return (
-    <Box width={BlockSize.Full} style={{ margin: '0 -8px' }}>
-      <ConfirmInfoRow label={`${t('primaryType')}:`}>
+    <Box width={BlockSize.Full}>
+      <ConfirmInfoRow
+        label={`${t('primaryType')}:`}
+        style={{ paddingLeft: 0, paddingRight: 0 }}
+      >
         <ConfirmInfoRowText text={primaryType} />
       </ConfirmInfoRow>
-      <Box style={{ margin: '0 -8px' }}>
+      <Box style={{ marginLeft: -8 }}>
         <DataTree data={sanitizedMessage.value} />
       </Box>
     </Box>
```

Comment:
`ConfirmInfoRow` applies padding on `8px` to the row on each side, thus I had to do some style tweaks to achieve tree structure as in the designs.